linchpin/api/context.py

```diff
@@ -53,55 +60,52 @@ def load_config(self, lpconfig=None):
         self.cfgs = {}
 
         expanded_path = None
-        config_found = False
 
         if lpconfig:
             CONFIG_PATH = [ lpconfig ]
-        else:
+        else: 
             # simply modify this variable to adjust where linchpin.conf can be found
-            CONFIG_PATH = [
-                '{0}/linchpin.conf'.format(self.lib_path)
-            ]
-
+            CONFIG_PATH = [ 
+                '{0}/linchpin.conf'.format(self.lib_path),
+                '/etc/linchpin.conf',
+                '~/.config/linchpin/linchpin.conf',
+                '{0}/linchpin.conf'.format(self.workspace) 
+                #self.workspace is set in runcli beforehand, will never be None
+            ]   
+        existing_paths = []
         for path in CONFIG_PATH:
-            expanded_path = (
+            expanded_path = ( 
                 "{0}".format(os.path.realpath(os.path.expanduser(path))))
 
             # implement first found
             if os.path.exists(expanded_path):
                 # logging before the config file is setup doesn't work
                 # if messages are needed before this, use print.
-                config_found = True
-                break
-
-        if not config_found:
+                existing_paths.append(expanded_path)
+        if len(existing_paths) is 0:
```

Comment:
as you are comparing values, I think it should be  len(existing_paths) == 0 rather than len(existing_paths) is 0
It might create problems 
Refer : https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is-in-python